src/aria2.h

```diff
@@ -37,6 +37,7 @@ class Aria2
 
     std::string addUri(const std::vector<std::string>& uri, const std::vector<std::pair<std::string, std::string>>& options = {});
     std::string tellStatus(const std::string& gid, const std::vector<std::string>& statusKey);
+    std::string getNewRpcSecret();
```

Comment:
Declare this as a static member function

Reply:
done